docker/Dockerfile

```diff
@@ -19,6 +19,7 @@ ARG PYTHON_DEPS=""
 ARG SYSTEM_DEPS=""
 ARG INDEX_URL=""
 ENV AIRFLOW_HOME=${AIRFLOW_USER_HOME}
+ENV PATH="/usr/local/airflow/.local/bin:/root/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/airflow/.local/lib/python3.7/site-packages:/usr/local/airflow/.local/bin:/usr/local/airflow/.local/lib/python3.7/site-packages:/usr/local/airflow/.local/bin:/usr/local/airflow/.local/lib/python3.7/site-packages"
```

Comment:
There are more paths here than I would have expected.

You're also completely overwriting `PATH`. Usually you'd just update/append to it with the specific paths you want to add:
`ENV PATH="$PATH:/usr/local/airflow/.local/lib/python3.7/site-packages:..."`
I think some of the paths you're adding here should be in PATH by default: